msg/vehicle_odometry.msg

```diff
@@ -1,68 +1,60 @@
 # Vehicle odometry data. Fits ROS REP 147 for aerial vehicles
 uint64 timestamp		# time since system start (microseconds)
-
 uint64 timestamp_sample
 
-# Covariance matrix index constants
-uint8 COVARIANCE_MATRIX_X_VARIANCE=0
-uint8 COVARIANCE_MATRIX_Y_VARIANCE=6
-uint8 COVARIANCE_MATRIX_Z_VARIANCE=11
-uint8 COVARIANCE_MATRIX_ROLL_VARIANCE=15
-uint8 COVARIANCE_MATRIX_PITCH_VARIANCE=18
-uint8 COVARIANCE_MATRIX_YAW_VARIANCE=20
-uint8 COVARIANCE_MATRIX_VX_VARIANCE=0
-uint8 COVARIANCE_MATRIX_VY_VARIANCE=6
-uint8 COVARIANCE_MATRIX_VZ_VARIANCE=11
-uint8 COVARIANCE_MATRIX_ROLLRATE_VARIANCE=15
-uint8 COVARIANCE_MATRIX_PITCHRATE_VARIANCE=18
-uint8 COVARIANCE_MATRIX_YAWRATE_VARIANCE=20
-
 # Position and linear velocity frame of reference constants
-uint8 LOCAL_FRAME_NED=0         # NED earth-fixed frame
-uint8 LOCAL_FRAME_FRD=1         # FRD earth-fixed frame, arbitrary heading reference
-uint8 LOCAL_FRAME_OTHER=2       # Not aligned with the std frames of reference
-uint8 BODY_FRAME_FRD=3          # FRD body-fixed frame
+uint8 FRAME_UNKNOWN     = 0
+uint8 LOCAL_FRAME_NED   = 1 # NED earth-fixed frame
+uint8 LOCAL_FRAME_FRD   = 2 # FRD earth-fixed frame, arbitrary heading reference
+uint8 BODY_FRAME_FRD    = 3 # FRD body-fixed frame
```

Comment:
This one only makes sense for velocity right?

Reply:
Yes, should we split them for velocity and position? That's actually what I was already doing in EKF backend (followup PR coming).

Reply:
yeah probably

Reply:
`POSE_FRAME_NED`, etc.